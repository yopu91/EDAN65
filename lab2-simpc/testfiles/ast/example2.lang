int main(int a, int b){
	int d=3;
	if(a!=b){
		int d=a;
		return d;
	}else{
		return d;
	}

b=0;
return d;

}
