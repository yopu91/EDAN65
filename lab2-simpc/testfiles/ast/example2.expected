Program
  List
    ParamFunc
      IdDecl
      List
        Param
          IdDecl
        Param
          IdDecl
      List
        Declare
          IdDecl
          Opt
            Numeral
        IfStmt
          Neq
            IdUse
            IdUse
          List
            Declare
              IdDecl
              Opt
                IdUse
            ReturnStmt
              IdUse
        Assignment
          IdUse
          Numeral
        ReturnStmt
          IdUse
